### PR TITLE
Add documentation about the new values in trigger auth in PostgreSQL Scaler

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: jorturfer <jorge_turrado@hotmail.es>

Add documentation about the new values in trigger auth in PostgreSQL Scaler

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Related https://github.com/kedacore/keda/pull/2114
Fixes https://github.com/kedacore/keda/issues/2110

### CODE REVIEW

File: content/docs/2.5/scalers/postgresql.md [hunk 1]
```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
 
 **Password Authentication:**
 
-- `password` - Password for configured user to login to postgreSQL database
-variables.
+- `host:` Service URL to postgresql. Note that you should use a full svc URL as KEDA will need to contact postgresql from a different namespace
```
Comment (arschles) [inline content/docs/2.5/scalers/postgresql.md:78]:
```suggestion
- `host:` Service URL to postgresql. Note that you should use a fully qualified URL -- including the namespace -- as KEDA will need to contact postgresql from a different namespace
```

**Review by arschles — CHANGES_REQUESTED** [review]

File: content/docs/2.5/scalers/postgresql.md (hunk 1 above)
Comment (JorTurFer) [reply content/docs/2.5/scalers/postgresql.md:78]: Honestly, I have copied and pasted from the current "metadata definitions", so I will change both :)

File: content/docs/2.5/scalers/postgresql.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Alternatively, a user can specify individual
 arguments (host, userName, password, etc.), and the scaler will form a connection string
 internally.
 
-- `host:` Service URL to postgresql. Note that you should use a full svc URL as KEDA will need to contact postgresql from a different namespace
+- `host:` Service URL to postgresql. Note that you should use a fully qualified URL -- including the namespace -- as KEDA will need to contact postgresql from a different namespace.
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:25]:
```suggestion
- `host:` Service URL to PostgreSQL. Note that you should use a fully qualified URL (including the namespace) as KEDA will need to contact PostgreSQL from a different namespace.
```

File: content/docs/2.5/scalers/postgresql.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Alternatively, a user can specify individual
 arguments (host, userName, password, etc.), and the scaler will form a connection string
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:22]: Nit - Can you ensure all values use the `PostgreSQL`  casing please given it's a product name?

File: content/docs/2.5/scalers/postgresql.md [hunk 4]
```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
 
 **Password Authentication:**
 
-- `password` - Password for configured user to login to postgreSQL database
-variables.
+- `host:` Service URL to postgresql. Note that you should use a fully qualified URL -- including the namespace -- as KEDA will need to contact postgresql from a different namespace.
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:78]: See above

File: content/docs/2.5/scalers/postgresql.md [hunk 5]
```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
...
+- `userName:` Username for postgresql user
+- `password` Password for configured user to login to postgreSQL database variables.
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:80]:
```suggestion
- `password`: Password for configured user to login to postgreSQL database.
```

File: content/docs/2.5/scalers/postgresql.md [hunk 6]
```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
...
+- `port:` Postgresql port
+- `dbName:` Postgresql Database name
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:82]:
```suggestion
- `dbName`: Name of PostgreSQL database
```

File: content/docs/2.5/scalers/postgresql.md [hunk 7]
```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
 
 **Password Authentication:**
 
-- `password` - Password for configured user to login to postgreSQL database
-variables.
+- `host:` Service URL to postgresql. Note that you should use a fully qualified URL -- including the namespace -- as KEDA will need to contact postgresql from a different namespace.
+- `userName:` Username for postgresql user
+- `password` Password for configured user to login to postgreSQL database variables.
+- `port:` Postgresql port
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:81]:
```suggestion
- `port`: Port of PostgreSQL database
```

File: content/docs/2.5/scalers/postgresql.md [hunk 8]
```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
 
 **Password Authentication:**
 
-- `password` - Password for configured user to login to postgreSQL database
-variables.
+- `host:` Service URL to postgresql. Note that you should use a fully qualified URL -- including the namespace -- as KEDA will need to contact postgresql from a different namespace.
+- `userName:` Username for postgresql user
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:79]:
```suggestion
- `userName`: Username for postgresql user
```

File: content/docs/2.5/scalers/postgresql.md (hunk 4 above)
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:78]:
```suggestion
- `host`: Service URL to postgresql. Note that you should use a fully qualified URL -- including the namespace -- as KEDA will need to contact postgresql from a different namespace.
```

File: content/docs/2.5/scalers/postgresql.md [hunk 9]
```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
...
+- `password` Password for configured user to login to postgreSQL database variables.
+- `port:` Postgresql port
+- `dbName:` Postgresql Database name
+- `sslmode:` SSL policy for communicating with database
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/postgresql.md:83]:
```suggestion
- `sslmode`: SSL policy for communicating with database
```

Is the field `sslmode` or `sslMode`?

File: content/docs/2.5/scalers/postgresql.md (hunk 3 above)
Comment (tomkerkhove) [reply content/docs/2.5/scalers/postgresql.md:22]: Same for the `<name>:` where the `:` needs to be moved out.

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Just needs a few improvements

File: content/docs/2.5/scalers/postgresql.md (hunk 3 above)
Comment (tomkerkhove) [reply content/docs/2.5/scalers/postgresql.md:22]: Actually let's align with the majority of scalers:
> - `host` - The hostname of the MSSQL instance endpoint.

Comment (JorTurFer) [reply content/docs/2.5/scalers/postgresql.md:22]: Do you mean using `propery - description`? Prometheus Scaler for example uses `property: description` also. WDYT if in another PR I review the scalers docs and unify the notation?

File: content/docs/2.5/scalers/postgresql.md (hunk 9 above)
Comment (JorTurFer) [reply content/docs/2.5/scalers/postgresql.md:83]: In the connection string is `sslMode` but our trigger variable is [`sslmode`](https://github.com/kedacore/keda/blob/main/pkg/scalers/postgresql_scaler.go#L105)

Comment (tomkerkhove) [reply content/docs/2.5/scalers/postgresql.md]: Can you elaborate on that please? You mean trigger metadata uses `sslMode` but trigger authentication uses `sslmode`?

Comment (tomkerkhove) [reply content/docs/2.5/scalers/postgresql.md]: No, the other way around probably?

File: content/docs/2.5/scalers/postgresql.md (hunk 3 above)
Comment (tomkerkhove) [reply content/docs/2.5/scalers/postgresql.md:22]: Yes, that notation.

A seperate PR would be awesome, if you are willing to do that of course.

Comment (JorTurFer) [reply content/docs/2.5/scalers/postgresql.md:22]: Sure, I will do :)

File: content/docs/2.5/scalers/postgresql.md (hunk 9 above)
Comment (JorTurFer) [reply content/docs/2.5/scalers/postgresql.md]: The required key in the connection string is `sslMode`, but inside the scaler we are looking for `sslmode` when we are parsing the metadata. Our key is `sslmode`, we take the value from the metadata key `sslmode` and place in the connection string key `sslMode`

Comment (JorTurFer) [reply content/docs/2.5/scalers/postgresql.md]: Maybe we can look for both when we are parsing the scaler metadata and deprecate `sslmode` in favor of `sslMode` (the last one is how we have to place in connection string)

Comment (tomkerkhove) [reply content/docs/2.5/scalers/postgresql.md]: If `sslMode` is only in the connection string then we can just keep on using `sslmode` as we do today, no need to check for `sslMode` field then as well

Comment (JorTurFer) [reply content/docs/2.5/scalers/postgresql.md]: okey, so basically we can let it as it is :)

File: content/docs/2.5/scalers/postgresql.md (hunk 3 above)
Comment (JorTurFer) [reply content/docs/2.5/scalers/postgresql.md:22]: I have opened it
https://github.com/kedacore/keda-docs/discussions/539

**Review by zroubalik — APPROVED** [review]

**Pull request merged**